src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -231,6 +242,7 @@ namespace winrt::TerminalApp::implementation
 
             // Update the control to reflect the changed title
             _headerControl.Title(activeTitle);
+            _SetToolTip(_GetActiveTitle());

```

Comment:
Are we sure this is called at all the right times? How about during initialization when we set the first title